### DISCUSSION TITLE
Source of the complete project of the Windows version of Freetube

### DISCUSSION THREAD

**Discussion by fbricart** [FreeTubeApp/FreeTube#2002] [General]
Hi,

I've been trying to find the source of the complete project of the Windows version of Freetube and I couldn't find it. Does anyone know if it's possible to find it somewhere and where is it?

Many thanks in advance.

Regards,
F.

**Comment by PrestonN** [comment]
You're already in the right place. It's right here, on this GitHub repository.

**Reply by fbricart** [reply]
Thanks for your answer. Maybe I'm trying to use a wrong IDE. I'm not used to this language and I'm trying to get inside this project to understand how it goes.

Which IDE should I use to work with this project and to compile it properly?

**Reply by PrestonN** [reply]
The project is written in JavaScript (specifically, Vue.js), so there isn't a dedicated IDE for working with it. You will need a text editor of your choice to get started. Popular ones are VSCode, Atom, and Sublime, though there are a few others out there as well. You can find more information on setting up a dev environment over on our [Getting Started Page](https://docs.freetubeapp.io/development/getting-started/).

**Reply by fbricart** [reply]
Thanks again for your answer! With the info you've given to me I'll be able to find out some more info to get started with this kind of coding.

Cheers!
